src/pages/MyFiles.vue

```diff
@@ -27,12 +27,14 @@ export default {
   name: "MyFiles",
   setup() {
     const store = useStore();
-    const { fetchStorageLimits, fetchMyFiles } = useArcanaStorage();
+    const { initStorage, fetchStorageLimits, fetchMyFiles } =
+      useArcanaStorage();
 
     const files = computed(() => store.getters.myFiles);
 
     onMounted(async () => {
       document.title = "My Files | Arcana Demo";
+      await initStorage();
```

Comment:
Had to move the storage SDK initialisation from App.vue to MyFiles.vue and SharedWithMe.vue. We can't show a blocking loader since the new `Storage.init` requires the user to approve a request on the wallet.